### ISSUE TITLE
PLEASE RELEASE

### ISSUE THREAD

**Issue by anandakrishnas** [karosium/smbusb#33]
Add precompiled files for cp2112. so i can directly use with my cp2112.

thanks

**Comment by othello777** [comment]
I dont think this project supports the cp2112. you'll need to buy a FX2LP if you want to use the code unmodified. it's only around $10
